app/views/waste_carriers_engine/conviction_details_forms/new.html.erb

```diff
@@ -18,6 +20,7 @@
         <% if @conviction_details_form.errors[:position].any? %>
         <span class="error-message"><%= @conviction_details_form.errors[:position].join(", ") %></span>
         <% end %>
+        <p><%= t(".paragraph_1") %></p>
```

Comment:
I think this may have been added to the wrong spot – it's been included in the position fieldset, rather than directly under the h1.